conf/st2.conf.sample

```diff
@@ -73,8 +73,12 @@ backend = flat_file
 home_dir = /opt/cslang
 
 [content]
+# A URL pointing to the pack index. StackStorm Exchange is used by default. Use a comma-separated list for multiple indexes if you want to get other packs discovered with "st2 pack search".
+index_url = https://index.stackstorm.org/v1/index.json # comma separated list allowed here.
```

Comment:
Yep. And I think left-to-right is probably better for consistency since we already do this in other places - where you specify multiple MongoDB and RabbitMQ URLs, etc.